repos/repos.go

```diff
@@ -32,6 +32,7 @@ type RepoList interface {
 type TrackedRepository struct {
 	Owner              string `json:"owner"`
 	Name               string `json:"name"`
+	DefaultBranch      string `json:"default_branch"`
```

Comment:
Why is this snake_case when others are camelCase?

Reply:
That's how it is exported from other services. Will file a separate PR to make sure they are all consistent